app.py

```diff
@@ -27,6 +29,9 @@
 
 
 def verify_token(token: str):
+    match=re.search('^[A-Za-z0-9]+$', token)
```

Comment:
Put spaces next to the equals sign. Like this `match = re.search(...)`
To match the style of the rest of the code